src/Runner/Parallel/ParallelConfig.php

```diff
@@ -19,8 +19,8 @@
  */
 final class ParallelConfig
 {
-    private const DEFAULT_FILES_PER_PROCESS = 10;
-    private const DEFAULT_PROCESS_TIMEOUT = 120;
+    public const DEFAULT_FILES_PER_PROCESS = 10;
+    public const DEFAULT_PROCESS_TIMEOUT = 120;
```

Comment:
```suggestion
    /** @internal */
    public const DEFAULT_FILES_PER_PROCESS = 10;
    /** @internal */
    public const DEFAULT_PROCESS_TIMEOUT = 120;
```

Reply:
I thought about making it internal, but then I thought that maybe it's not that bad to allow it to use it in the configs if someone wants to do it?

Reply:
I prefer to keep BC promise minimal, let's make it internal and open if someone would request it.